tt_eager/tt_lib/csrc/operations/primary/module.hpp

```diff
@@ -726,6 +728,32 @@ void py_module(py::module& m_primary) {
         py::arg("compute_kernel_config").noconvert() = std::nullopt,
         "Performs a nll_loss_backward operation. Returns an input_grad tensor.");
 
+    // moreh_nll_loss_unreduced
+    m_primary.def(
+        "moreh_nll_loss_unreduced",
+        &moreh_nll_loss_unreduced,
+        py::arg("input_tensor").noconvert(),
+        py::arg("target_tensor").noconvert(),
+        py::arg("weight_tensor").noconvert() = std::nullopt,
+        py::arg("output_tensor").noconvert() = std::nullopt,
+        py::arg("ignore_index").noconvert(),
+        py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
```

Comment:
As we are merging things to ttnn, we are aligning things between libs. In a couple months we expect that all usages of output_mem_config are replaced with `memory_config`, and we keep it as a py::kw_only argument instead of positional and use default none, to let the op potentially choose input's tensor memory config for output.

I am not insisting on a change here, but if we align it now, we have couple less ops and usages in future.

Reply:
I attempted to make the changes you suggested, but it turned out to be more work than expected. `moreh_nll_loss_unreduce` calls `moreh_nll_loss`, which in turn calls `moreh_sum`.

For now, I will just rename it to `memory_config` and address the actual functionality changes in a later issue, where related operations will be updated together.